platforms/web/src/App.test.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2023 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import App from './App';
+
+test('User can enter an umlaut using IME type input', async () => {
+    const user = userEvent.setup();
+
+    render(<App />);
+    const textbox = screen.getByRole('textbox');
+    await user.type(textbox, '{Shift>}^{/Shift}a');
+    screen.debug();
```

Comment:
This will be implemented/expanded if I can figure out how to get it working in the tests